packages/core/upload/admin/src/pages/App/MediaLibrary.js

```diff
@@ -199,7 +199,9 @@ export const MediaLibrary = () => {
         />
 
         <ContentLayout>
-          {selected.length > 0 && <BulkActions selected={selected} onSuccess={selectAll} />}
+          {selected.length > 0 && (
+            <BulkActions parentFolder={currentFolder} selected={selected} onSuccess={selectAll} />
```

Comment:
It feels a bit weird to me to pass `currentFolder` as `parentFolder`. What do you think about calling the prop `folder` or `currentFolder`?